README.md

```diff
@@ -1,2 +1,50 @@
-# My-Portfolio-1
-This project is to create the first page of my own portfolio.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My Portfolio
+
+> This is a html page which is contains information about me.
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
```

Comment:
Kindly add the steps on how can someone run this project. For example, how to clone this project and run it on a local machine.